test/functional/test-performance.js

```diff
@@ -195,9 +195,9 @@ describe('performance', () => {
         expect(perf.events_.length).to.equal(2);
 
         return perf.coreServicesAvailable().then(() => {
-          expect(flushSpy.callCount).to.equal(2);
+          expect(flushSpy.callCount).to.equal(3);
```

Comment:
I suspect the same. loadPromise never fired before. Since `ol`  is marked in constructor, it happens for all tasks?